Content.Shared/Weather/WeatherComponent.cs

```diff
@@ -12,7 +11,7 @@ public sealed partial class WeatherComponent : Component
     /// <summary>
     /// Currently running weathers
     /// </summary>
-    [ViewVariables, DataField("weather", customTypeSerializer:typeof(PrototypeIdDictionarySerializer<WeatherData, WeatherPrototype>))]
+    [DataField(customTypeSerializer:typeof(PrototypeIdDictionarySerializer<WeatherData, WeatherPrototype>))]
```

Comment:
use protoid

Reply:
when I tried to do it, it gave me a lot of errors. WeatherData is not a prototype. How do I do this?

Reply:
WeatherPrototype is the prototype, so do `Dictionary<ProtoId<WeatherPrototype>, WeatherData>`

Reply:
oh